Backend/Middleweres/Authentication.js

```diff
@@ -1,30 +0,0 @@
-// write authentication middlewere here
-// create a seperate file for each middlewere eg. Validator.js , Autorisation.js etc
-
-
-
-const jwt = require('jsonwebtoken');
-require('dotenv').config();
-
-const authentication = (req, res, next) => {
-    if(!req.headers.authorization){
-        return res.send("Not authorized")
-    }
-    const user_token = req.headers.authorization;
-    // console.log('user_token:', user_token);
-    
-    jwt.verify(user_token, process.env.SECRET_KEY, function(err, decoded) {
-        if(err){
-            return res.send("Please login again")
-        }
-        console.log(decoded)
```

Comment:
suggestion: don't print the decoded info in the console. It may also contain some other user information.